lib/Service/FollowService.php

```diff
@@ -27,45 +27,57 @@
  *
  */
 
+namespace OCA\Social\Service;
 
-namespace OCA\Social\Service\ActivityPub;
 
-
-use Exception;
+use daita\MySmallPhpTools\Exceptions\MalformedArrayException;
+use daita\MySmallPhpTools\Traits\TArrayTools;
 use OCA\Social\Db\FollowsRequest;
-use OCA\Social\Exceptions\ActorDoesNotExistException;
 use OCA\Social\Exceptions\CacheActorDoesNotExistException;
 use OCA\Social\Exceptions\FollowDoesNotExistException;
 use OCA\Social\Exceptions\FollowSameAccountException;
+use OCA\Social\Exceptions\InvalidOriginException;
 use OCA\Social\Exceptions\InvalidResourceException;
+use OCA\Social\Exceptions\RedundancyLimitException;
+use OCA\Social\Exceptions\Request410Exception;
 use OCA\Social\Exceptions\RequestException;
 use OCA\Social\Exceptions\SocialAppConfigException;
+use OCA\Social\Exceptions\UnknownItemException;
 use OCA\Social\Exceptions\UrlCloudException;
-use OCA\Social\Model\ActivityPub\ACore;
-use OCA\Social\Model\ActivityPub\Activity\Accept;
-use OCA\Social\Model\ActivityPub\Activity\Reject;
-use OCA\Social\Model\ActivityPub\Activity\Undo;
-use OCA\Social\Model\ActivityPub\Follow;
+use OCA\Social\Model\ActivityPub\Activity\Follow;
+use OCA\Social\Model\ActivityPub\Actor\Person;
 use OCA\Social\Model\ActivityPub\OrderedCollection;
-use OCA\Social\Model\ActivityPub\Person;
 use OCA\Social\Model\InstancePath;
-use OCA\Social\Service\ActivityService;
-use OCA\Social\Service\ConfigService;
-use OCA\Social\Service\MiscService;
 
+class FollowService {
+
+
+	use TArrayTools;
+
+
+	const REQUEST_INBOX = 1;
```

Comment:
Can we unify this with the constants from ActivityService? We have the same ones over there.

Reply:
fixed